lib/linguist/heuristics.yml

```diff
@@ -216,6 +216,13 @@ disambiguations:
     pattern: '^<\?(?:php)?'
   - language: SourcePawn
     pattern: '^public\s+(?:SharedPlugin(?:\s+|:)__pl_\w+\s*=(?:\s*{)?|(?:void\s+)?__pl_\w+_SetNTVOptional\(\)(?:\s*{)?)'
+  - language: NASL
+    pattern:
+    - '^\s*include\s*\(\s*(?:"|'')[\\/\w\-\.:\s]+(?:"|'')\s*\)\s*;'
+    - '^\s*(?:global|local)_var\s+(?:[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)(?:,\s*[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)*+\s*;'
```

Comment:
This line contains capturing-groups with empty expressions: `|)`. If you intended to make these matches optional, use `)?` instead:

~~~diff
-'^\s*(?:global|local)_var\s+(?:[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)(?:,\s*[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)*+\s*;'
+'^\s*(?:global|local)_var\s+(?:[\w\-]+(?:\s*=\s*[\w\-"'']+)?\s*)(?:,\s*[\w\-]+(?:\s*=\s*[\w\-"'']+)?\s*)*+\s*;'
~~~

Reply:
More elegant indeed. Changed in latest commit.